portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsPage.testcase

```diff
@@ -564,6 +564,72 @@ definition {
 		}
 	}
 
+	@description = "Assert that is possible to publish Publication with Asset Publisher configurations"
+	@priority = 3
+	test CanPublishWithAssetPublisherConfigurations {
+		task ("Create two widget pages and add to each page an Asset Publisher") {
+			for (var pageNumber : list "1,2") {
+				JSONLayout.addPublicLayout(
+					groupName = "Guest",
+					layoutName = "Widget Page Name ${pageNumber}");
+
+				JSONLayout.addWidgetToPublicLayout(
+					groupName = "Guest",
+					layoutName = "Widget Page Name ${pageNumber}",
+					widgetName = "Asset Publisher");
+			}
+		}
+
+		task ("Add new Publication and use it") {
+			JSONPublications.addPublication(publicationName = "Publication Name");
+
+			Publications.selectPublication(publicationName = "Publication Name");
+		}
+
+		task ("Go to Widget Page Name 1 and configure Asset Publisher") {
+			Navigator.gotoPage(pageName = "Widget Page Name 1");
+
+			AssetPublisherPortlet.configureManualAssetSelectionPG();
+
+			Alert.viewSuccessMessage();
+		}
+
+		task ("Go to Widget Page Name 2 and configure Asset Publisher") {
+			Navigator.gotoPage(pageName = "Widget Page Name 2");
+
+			AssetPublisherPortlet.configureManualAssetSelectionPG();
+
+			Alert.viewSuccessMessage();
+		}
+
+		task ("Assert that the changes were made") {
+			PublicationsNavigator.gotoReviewChanges(
+				gotoPublicationsAdmin = "true",
+				publicationName = "Publication Name");
+
+			PublicationsChanges.viewChanges(
+				changeCategory = "Page",
+				changeSite = "Guest",
+				changeTitle = "Widget Page Name 1",
+				changeType = "Modified",
+				changeUser = "Test Test");
+
+			PublicationsChanges.viewChanges(
+				changeCategory = "Page",
+				changeOrder = 2,
+				changeSite = "Guest",
+				changeTitle = "Widget Page Name 2",
+				changeType = "Modified",
+				changeUser = "Test Test");
+		}
+
+		task ("Publish the publication and assert that it was published successfully") {
+			Publications.publishPublication(
+				publicationName = "Publication Name",
+				reviewChangesPage = "true");
```

Comment:
Should be the following due to changes we made in [LPS-190513](https://liferay.atlassian.net/browse/LPS-190513)
```
Publications.publishPublication(
	gotoPublicationsAdmin = "true",
	publicationName = "Publication Name");
```